_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -548,8 +548,29 @@ protected function test__sync_health() {
 				);
 
 			} elseif ( Sync_Health::get_status() === Sync_Health::STATUS_OUT_OF_SYNC ) {
-
-				// Sync has experienced Data Loss.
+				/*
+				 * Sync has experienced Data Loss.
+				 */
+
+				$description  = '<p>';
+				$description .= esc_html__( 'The information synced by Jetpack ensures that Jetpack Search, Related Posts and other features are aligned with your site’s current content.', 'jetpack' );
+				$description .= '</p>';
+				$description .= '<p>';
+				$description .= sprintf(
+					'<span class="dashicons fail"><span class="screen-reader-text">%1$s</span></span> ',
+					esc_html__( 'Error', 'jetpack' )
+				);
+				$description .= wp_kses(
+					__( 'Jetpack has detected an error while syncing your site. <strong>We recommend <a id="full_sync_request_link" href="#">a full sync</a> to align Jetpack with your site data.</strong>', 'jetpack' ),
```

Comment:
@mdbitz Is this link supposed to lead somewhere special?

Reply:
its utilized in a javascript handler, so no :)